core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -314,6 +314,14 @@ public enum Property {
       "Enables additional JVM metrics collection and reporting using Micrometer. Requires "
           + "property 'general.micrometer.enabled' to be set to 'true' to take effect.",
       "2.1.0"),
+  GENERAL_MICROMETER_LOG_METRICS_ENABLED("general.micrometer.log.metrics.enabled", "false",
```

Comment:
instead of having two properties, could have a single property with three values. One value for no logging, one for log4j and one fro logback.